deployer/hub.py

```diff
@@ -100,7 +101,7 @@ def __init__(self, cluster, spec):
         self.cluster = cluster
         self.spec = spec
 
-    def get_generated_config(self, auth_provider, secret_key):
+    def get_generated_config(self, auth_provider: KeyProvider, secret_key):
```

Comment:
Thinking about typing?

Reply:
haha yes, now that I'm on an editor  (vscode) that uses a proprietary (pylance) plugin to provide better autocomplete